frontend/src/Components/AlgInputs/AlgInputs.js

```diff
@@ -42,6 +42,7 @@ export default function AlgInputs (props) {
                 <AlgInputCustomized
                     category = { props.category }
                     visualizeResults={ (result) => visualizeResults(result) }
+                    loadingResults={ (bool) => props.loadingResults(bool)}
```

Comment:
naming this argument `bool` is confusing. Something more descriptive like `isLoading` would be better

Reply:
fixed